### PR TITLE
 Add cap tasks to list aliases and swap catalog aliases

### LINKED ISSUE #105: Automate updating aliases
http://localhost:9000/solr/admin/collections?action=CLUSTERSTATUS 
catalog-production is for the live version of the catalog index
catalog-rebuild is for the collection we use to reindex the catalog

### PR DESCRIPTION
**Pull request by eliotjordan**
Closes #105

### CODE REVIEW

File: config/deploy.rb [hunk 1]
```diff
@@ -65,6 +65,34 @@
...
+  task :list do
+    on roles(:main) do
+      execute "curl 'http://localhost:8983/solr/admin/collections?action=LISTALIASES'"
+    end
+  end
+
+  # Swaps the catalog-rebuild and catalog-production aliases.
+  # Production and rebuild collections are set with env variables when running the task.
+  task :catalog do
+    if ENV['production'] && ENV['rebuild']
```
Comment (tampakis) [inline config/deploy.rb]: Can you use capital names for the environment variables?

**Comment by eliotjordan** [comment]
@tampakis  Updated

File: config/deploy.rb [hunk 2]
```diff
@@ -65,6 +65,36 @@
...
+  task :catalog do
+    production = ENV['PRODUCTION']
+    rebuild = ENV['REBUILD']
+    if production && rebuild
+      on roles(:main) do
+        # Delete the rebuild alias
+        execute "curl ‘http://localhost:8983/solr/admin/collections?action=DELETEALIAS&name=catalog-rebuild’"
+
+        # Move the catalog-production alias
+        execute "curl 'http://lilb-solr.princeton.edu:8983/solr/admin/collections?action=CREATEALIAS&name=catalog-production&collections=#{production}'"
```
Comment (tampakis) [inline config/deploy.rb]: should be localhost:8983?

File: config/deploy.rb [hunk 3]
```diff
@@ -65,6 +65,36 @@
...
+
+        # Add the rebuild alias to its new location
+        execute "curl ‘http://lilb-solr.princeton.edu:8983/solr/admin/collections?action=CREATEALIAS&name=catalog-rebuild&collections=#{rebuild}’"
```
Comment (tampakis) [inline config/deploy.rb]: i think the single quote got changed to a different character

File: config/deploy.rb (hunk 2 above)
Comment (eliotjordan) [reply config/deploy.rb]: yep. good catch.

**Comment by eliotjordan** [comment]
@tampakis Should be good now.

**Pull request merged**